src/bin/cargo/commands/rustc.rs

```diff
@@ -12,7 +12,7 @@ pub fn cli() -> Command {
         .arg(
             Arg::new("args")
                 .num_args(0..)
-                .help("Rustc flags")
+                .help("Extra rustc flags")
```

Comment:
The `after_help` for `cargo test` also says

```
Run `cargo test -- --help` for test binary options.
```

Should we include that?  For `run` it is more complicated because the command isn't guaranteed to have a `--help`.

Reply:
I've considered that. It might be misleading so I probably avoid that.

Reply:
Sorry if I wasn't clear, I was specifically calling this out on rustc's help as rustc and rustdoc shouldn't have the same problem as `run`

Reply:
Good call. Just tried it but it's unfortunate. Running `cargo rustc -- --help` looks like this if you have multiple Cargo targets:

```console
$ cargo rustc -- --help
error: extra arguments to `rustc` can only be passed to one target, consider filtering
the package by passing, e.g., `--lib` or `--bin NAME` to specify a single target
```

Reply:
Thats fine